### ISSUE TITLE
Reference learning resources per topic?

### ISSUE THREAD

**Issue by kahboom** [kahboom/competencies#2]
Would it make sense to add learning resources for each topic listed in each competency? For example, articles, videos, or digital books that are free.

**Comment by lhein** [comment]
yes, adding at least one link to each item makes perfect sense. That was the first thought I had when reading the document...."where are the links" :D
